numba_rvsdg/rendering/rendering.py

```diff
@@ -203,17 +258,42 @@ def render_branching_block(
         digraph.node(str(name), shape="rect", label=body)
 
     def view(self, name: str):
+        """Method used to view the current SCFG as an external graphviz generated
+        PDF file.
```

Comment:
Arguments are missing.

Reply:
Fixed in [3e369c9](https://github.com/numba/numba-rvsdg/pull/59/commits/3e369c985b3af41872bcee5df9345852d780053f)